src/ui/invisible_window.h

```diff
@@ -22,31 +22,7 @@
 
 HWND create_invisible_window(LPCTSTR lpszClassName,
```

Comment:
The idea was to not pollute opensc.h with windows specific functions, that's why create_invisible_window was intended as header implementation only to be included in both, opensc-notify.c and notify.c (as part of libnotify/libopensc).

I think you can fix the compilation error by just adding "`static`" to create_invisible_window without any other modification.